src/System.Runtime.Extensions/tests/VoidMainWithExitCodeApp/project.json

```diff
@@ -7,6 +7,7 @@
     "System.Threading.Thread": "4.3.0-beta-devapi-24503-01"
   },
   "frameworks": {
+    ".NETCoreApp,Version=v1.0": {},
```

Comment:
Actually more likely a missing configuration for NuGetTargetMoniker in the CSProj.